### ISSUE TITLE
Parse error - national number incorrect for some phone numbers.

### ISSUE THREAD

**Issue by mattwarren1234** [ttacon/libphonenumber#19]
If I call
```
_, err := libphonenumber.Parse("15167706076", "US")
nationalNumber := num.GetNationalNumber()
```
National number *should* equal 5167706076...but instead it has a leading 1 - so it is 15167706076.

I looked into the parse code a bit but couldn't find any clear reason as to why this is occurring.

**Comment by mattwarren1234** [comment]
For added info:
`libphonenumber.IsValidNumber(num)`
returns true, and `err` is nil

**Comment by loopfz** [comment]
This should be fixed by #20

**Comment by ttacon** [comment]
Provisionally closing - @mattwarren1234 reopen this issue if it's still a problem.
